src/components/Inputs/Input.jsx

```diff
@@ -4,24 +4,29 @@ import TextField from '@mui/material/TextField';
 import { useStyles } from './InputStyle';
 
 
-export const Input = ({label, type='text'}) => {
+export const Input = ({ label, type='text' }) => {
     const classes = useStyles();
-    const [data, setData] = useState('');
+    const [value, setValue] = useState('');
```

Comment:
don't use useState () here, pass value and setValue as props because value will be used in the parent component

Reply:
fixed